server/character.lua

```diff
@@ -8,26 +8,9 @@ end
 
 ---@param source Source
 local function giveStarterItems(source)
-    local Player = QBCore.Functions.GetPlayer(source)
-
+    exports['um-idcard']:CreateMetaLicense(source, {'id_card', 'driver_license'})
```

Comment:
I'm struggling to understand what benefits that solution has over consuming the playerCreation event in qbx-cityhall?

Reply:
https://github.com/Qbox-project/qbx-core/blob/main/client/character.lua#L206C5-L206C5

Reply:
Oh, I see, but that would still be using the core if I were to add the export there, unless I am misunderstanding something

Reply:
By using core you mean consuming an event? That's ok. Events don't create dependencies. core is just telling anyone who cares to listen that a character was just created. If qbx-cityhall cares to listen, it can act on that event.